### ISSUE TITLE
Suggestion: Add option to sort by US State

### ISSUE THREAD

**Issue by arhasty** [sstangl/openpowerlifting#1189]
It would be great to have an option to select event location by state whenever a US fed is chosen. 
E.G. - 
I select "USAPL" and a second box or option allows me to then select USA-TX or USA-NY instead of showing results for all 50 states.

**Comment by sstangl** [comment]
That's a good idea. I think because of the way that would have to work, it's a feature best left to the new version of the site we're writing. The `meetlist.html` page is already so large that it takes browsers multiple seconds to perform table layout -- and to allow a state selector, we'd need even more CSS tags, and even more data sent.

But the beta server can send much less data, and therefore that won't be prohibitive.

**Comment by sstangl** [comment]
Also note that our list of states is in `tests/check-meet-csv`, in Python -- we'd probably want that list to be shared with Rust... Although the USA isn't likely to change states soon, so maybe it's OK to duplicate the code.

**Comment by arhasty** [comment]
Good point about the extra tags. That would definitely add to the data set. Side note - if you ever want/need any help,  I'd be happy to pitch in. Web dev isn't my strong suit, but I'm pretty good with SQL (and/or just doing grunt work). You guys are doing awesome work for the powerlifting community!
